task11/Shipment.ts

```diff
@@ -0,0 +1,105 @@
+import { AirEeastShipper, ChicagoSprintShipper, PacificParcelShipper, Shipper } from "./Shipper";
+import { State } from "./State";
+
+let uniqueId = 1;
+
+export enum ShipmentType {
+    Package,
+    Letter,
+    Oversize,
+}
+
+export interface IShipment {
+    // how to hide these props? currently only need in the Shipment class
+    state: State;
+    shipper: Shipper;
+    shipmentType: ShipmentType;
+
+    addMark(mark?: string): void;
+    ship(): string;
+    getShipmentId(): string
+}
+
+export const STANDARD_LETTER_MAX_WEIGHT = 15; // ounces
+export const STANDARD_PACKAGE_MAX_WEIGHT = 160; // ounces
```

Comment:
Please, put const under the import from the beginning.
All constant variables should come first :)